Core/MIPS/IR/IRCompLoadStore.cpp

```diff
@@ -46,113 +46,8 @@
-		int shiftReg = IRTEMP_3;
-
-		// addrReg = rs + imm
-		ir.Write(IROp::AddConst, addrReg, rs, ir.AddConstant(offset));
-		// shiftReg = (addr & 3) * 8
-		ir.Write(IROp::AndConst, shiftReg, addrReg, ir.AddConstant(3));
-		ir.Write(IROp::ShlImm, shiftReg, shiftReg, 3);
-		// addrReg = addr & 0xfffffffc (for stores, later)
-		ir.Write(IROp::AndConst, addrReg, addrReg, ir.AddConstant(0xFFFFFFFC));
-		// valueReg = RAM(addrReg)
-		ir.Write(IROp::Load32, valueReg, addrReg, ir.AddConstant(0));
-
-		switch (o) {
-		case 34: //lwl
-			// rt &= (0x00ffffff >> shift)
-			// Alternatively, could shift to a wall and back (but would require two shifts each way.)
-			ir.WriteSetConstant(maskReg, 0x00ffffff);
-			ir.Write(IROp::Shr, maskReg, maskReg, shiftReg);
-			ir.Write(IROp::And, rt, rt, maskReg);
-			// valueReg <<= (24 - shift)
-			ir.Write(IROp::Neg, shiftReg, shiftReg);
-			ir.Write(IROp::AddConst, shiftReg, shiftReg, ir.AddConstant(24));
-			ir.Write(IROp::Shl, valueReg, valueReg, shiftReg);
-			// rt |= valueReg
-			ir.Write(IROp::Or, rt, rt, valueReg);
-			break;
-		case 38: //lwr
-			// valueReg >>= shift
-			ir.Write(IROp::Shr, valueReg, valueReg, shiftReg);
-			// shiftReg = 24 - shift
-			ir.Write(IROp::Neg, shiftReg, shiftReg);
-			ir.Write(IROp::AddConst, shiftReg, shiftReg, ir.AddConstant(24));
-			// rt &= (0xffffff00 << (24 - shift))
-			// Alternatively, could shift to a wall and back (but would require two shifts each way.)
-			ir.WriteSetConstant(maskReg, 0xffffff00);
-			ir.Write(IROp::Shl, maskReg, maskReg, shiftReg);
-			ir.Write(IROp::And, rt, rt, maskReg);
-			// rt |= valueReg
-			ir.Write(IROp::Or, rt, rt, valueReg);
-			break;
-		case 42: //swl
-			// valueReg &= 0xffffff00 << shift
-			ir.WriteSetConstant(maskReg, 0xffffff00);
-			ir.Write(IROp::Shl, maskReg, maskReg, shiftReg);
-			ir.Write(IROp::And, valueReg, valueReg, maskReg);
-			// shiftReg = 24 - shift
-			ir.Write(IROp::Neg, shiftReg, shiftReg);
-			ir.Write(IROp::AddConst, shiftReg, shiftReg, ir.AddConstant(24));
-			// valueReg |= rt >> (24 - shift)
-			ir.Write(IROp::Shr, maskReg, rt, shiftReg);
-			ir.Write(IROp::Or, valueReg, valueReg, maskReg);
-			break;
-		case 46: //swr
-			// valueReg &= 0x00ffffff << (24 - shift)
-			ir.WriteSetConstant(maskReg, 0x00ffffff);
-			ir.Write(IROp::Neg, shiftReg, shiftReg);
-			ir.Write(IROp::AddConst, shiftReg, shiftReg, ir.AddConstant(24));
-			ir.Write(IROp::Shr, maskReg, maskReg, shiftReg);
-			ir.Write(IROp::And, valueReg, valueReg, maskReg);
-			ir.Write(IROp::Neg, shiftReg, shiftReg);
-			ir.Write(IROp::AddConst, shiftReg, shiftReg, ir.AddConstant(24));
-			// valueReg |= rt << shift
-			ir.Write(IROp::Shl, maskReg, rt, shiftReg);
-			ir.Write(IROp::Or, valueReg, valueReg, maskReg);
-			break;
-		default:
-			INVALIDOP;
-			return;
-		}
-
-		if (!load) {
-			// RAM(addrReg) = valueReg
-			ir.Write(IROp::Store32, valueReg, addrReg, ir.AddConstant(0));
-		}
+		// Should never get here, done in Comp_ITypeMem().
```

Comment:
Can't we just delete this whole function then?

Reply:
Ah, oops, I thought it was part of the jit interface or something.  Will remove in some hours.

-[Unknown]